velox/functions/prestosql/window/LeadLag.cpp

```diff
@@ -177,7 +176,7 @@ class LeadLagFunction : public exec::WindowFunction {
         }
```

Comment:
We can add this condition at line 173 and that handles this case for all cases (IGNORE NULLS or otherwise). It might be a simpler fix.

if (offset == 0) {
  rowNumbers_[i] = partitionOffset_ + i;
  continue;
}

Reply:
thank you @aditi-pandit for review , it makes sense to me. Addressed comment.